src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/results.html.twig

```diff
@@ -19,9 +19,24 @@
                 {% for elements in tokens %}
                     <tr>
                         <td><a href="{{ path('_profiler', { 'token': elements.token }) }}">{{ elements.token }}</a></td>
-                        <td>{{ elements.ip }}</td>
-                        <td>{{ elements.method }}</td>
-                        <td>{{ elements.url }}</td>
+                        <td>
+                            {{ elements.ip }}
+                            <a href="{{ path('_profiler_search_results', app.request.query.all|merge({'ip': elements.ip, 'token': elements.token})) }}">
```

Comment:
Does it make sense to include the token here? Wouldn't you usually want to filter just by the IP address?